sdk/python/feast/ui/yarn.lock

```diff
@@ -1345,10 +1345,10 @@
     minimatch "^3.1.2"
     strip-json-comments "^3.1.1"
 
-"@feast-dev/feast-ui@^0.20.2":
-  version "0.20.2"
-  resolved "https://registry.yarnpkg.com/@feast-dev/feast-ui/-/feast-ui-0.20.2.tgz#7af94f99f995bf6c0625ca504307fa2630f509e9"
-  integrity sha512-K5O5uTEkTtW3uuoI49YsZAutdv38T/psWygNcNpQ9UWMrlQCfHjMU0gTSPJpakVVIFoWRvn8C3Q4HN8tLB6ZAg==
+"@feast-dev/feast-ui@^0.20.3":
+  version "0.20.3"
```

Comment:
Why was this bumped manually?

Reply:
was accidentally missed in a previous PR